backend/pkg/backend/backend.go

```diff
@@ -101,10 +105,22 @@ func Run() {
 	dbConfig := createDatabaseConfig(config)
 	dbHandler := _database.Init(dbConfig)
 	dbHandler.StartReviewTableCleaner(globalCtx, time.Duration(config.DatabaseCleanerIntervalSec)*time.Second)
+	var clientset kubernetes.Interface
+	if config.K8sLocal {
```

Comment:
not sure how to do it... you suggest mocking the kubernetes interface?

Reply:
No. I am suggesting to create a clientset only if one of the FAKE_XX env vars was not set. same as we do for the monitor. If you need the clientset for the the coreModule (which I am not sure you do), maybe better to create an interface that wrap the clientset and with Enable and Disable implementation. Not completley sure it worth the effort, but if we want to be able to run w/o k8s we need to do it. 
@akpsgit @FrimIdan WDYT?

Reply:
The interface is too big to fake. The simplest thing will be on line 117 instead of logging fatal to log error and the clientset will be nil. Then each module that needs kubernetes client should have the responsibility to check if the kubernetes client is null and implement custom logic...

But anyway I think it's a good Idea to be able to configure kubernetes for out of cluster configs for local deployments and tests